### PR TITLE
Feature polygons

### PR DESCRIPTION
**Pull request by G0URAB**
Holes for Lau and nuts

### CODE REVIEW

File: src/main/java/org/dice_research/opal/launuts/ModelBuilder.java [hunk 1]
```diff
@@ -43,9 +56,93 @@
 		// Additional prefixes to reduce model size
 		model.setNsPrefix("laude", Vocabularies.NS_LAU_DE);
 		model.setNsPrefix("nutscode", Vocabularies.NS_EU_NUTS_CODE);
+
+		// Prefix for identifying a geometry type
+		model.setNsPrefix("sf", "http://www.opengis.net/ont/sf#");
+	}
+
+	public void addPolygons(Resource res, String resource_id, JSONArray nuts_or_lau_polygons, String resource_id_type) {
```
Comment (Aamir-Developer) [inline src/main/java/org/dice_research/opal/launuts/ModelBuilder.java:64]: In this function addPolygons,  multiple nested loops and conditional statements are present, which increases the overall run time complexity. Please check if it is possible to reduce it.

**Review by Aamir-Developer — COMMENTED** [review]
Exception handling is not implemented in most of the functions.

**Review by Aamir-Developer — COMMENTED** [review]
Testcases are not implemented to test the code.

**Comment by G0URAB** [comment]
This is a deprecated code. The logic of our parsers has been changed. We are not combining multi polygons any more. I will make a new pull request with new code.

**Pull request closed without merging**